arroyo-sql-macro/Cargo.toml

```diff
@@ -7,13 +7,18 @@ edition = "2021"
 proc-macro = true
 
 [dependencies]
+arroyo-api = { path = "../arroyo-api" }
```

Comment:
This dependency is a bit unfortunate, as now to build the macro we need to build essentially everything. It also doesn't appear that any imports from arroyo_api are used, so maybe this can be removed?

Reply:
Done